pkgs/development/compilers/v/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, fetchFromGitHub, glfw, freetype, curl }:
+
+let
+    # V compiler's source translated to C for bootstrapping.
+    vc = fetchurl {
+        url = "https://github.com/vlang/v/releases/download/v0.1.0/v.c";
```

Comment:
This does not seem to exist.

Reply:
Yes, seems like something is happening upstream.
Will change when upstream changes.